app/views/layouts/_head.html.haml

```diff
@@ -0,0 +1,22 @@
+/ We always want this rendered at the start.
+= stylesheet_link_tag "website", "data-turbo-track": "reload"
+
+- if landing_page?
+  / Get this rendered as the next thing
+  = javascript_include_tag('landing', type: :module, crossorigin: :anonymous, 'data-turbo-track': 'reload', 'data-turbo-eval': false)
```

Comment:
I'd move this back into `content_for :deferred` so it appears at the bottom of the `body` and doesn't block rendering.